### ISSUE TITLE
Archive this repo

### ISSUE THREAD

**Issue by ghinks** [pillarjs/templation#5]
@jonathanong  you have been identified as the last / most active committer in this repo which is inactive for a while. This ping is to check with you to determine:

- whether do you have any specific maintenance plans with this repo
- are you fine for this repo to be archived
- do you believe if there is another stake holder / affected party who should know

thanks!

**Comment by jonchurch** [comment]
@ghinks duplicate of #4 ?

**Comment by ghinks** [comment]
@jonchurch, yes I closed 4 and re-opened it as 5. Yesterday I had to thought made a mistake and templation was being used, it was not and so I opened another.

**Comment by jonathanong** [comment]
👍
